.github/workflows/pr-autopilot.yml

```
@@ -0,0 +1,163 @@
+        run: |
+          set -euo pipefail
+          python scripts/pr_autopilot_dispatch.py prepare-bundle \
+              --pr "${{ needs.triage.outputs.pr_number }}" \
+              --output /tmp/autopilot-bundle.json
+          echo "bundle_path=/tmp/autopilot-bundle.json" >> "$GITHUB_OUTPUT"
+
+      - name: Apply fixes via Claude
+        uses: anthropics/claude-code-action@v1
+        with:
+          anthropic_api_key: ${{ secrets.ANTHROPIC_API_KEY }}
+          mode: pr-fix
```

actionlint (1.7.12)

[error] input "mode" is not defined in action "anthropics/claude-code-action@v1". available inputs are "additional_permissions", "allowed_bots", "allowed_non_write_users", "anthropic_api_key", "assignee_trigger", "base_branch", "bot_id", "bot_name", "branch_name_template", "branch_prefix", "classify_inline_comments", "claude_args", "claude_code_oauth_token", "display_report", "exclude_comments_by_actor", "github_token", "include_comments_by_actor", "include_fix_links", "label_trigger", "path_to_bun_executable", "path_to_claude_code_executable", "plugin_marketplaces", "plugins", "prompt", "settings", "show_full_output", "ssh_signing_key", "track_progress", "trigger_phrase", "use_bedrock", "use_commit_signing", "use_foundry", "use_sticky_comment", "use_vertex"

(action)